src/config-import.js

```diff
@@ -17,6 +18,20 @@ const MAX_ICON_SIZE = 10_000_000
  *   fields:  { [id: string]: unknown }
  * }} PresetsFile
  */
+
+/**
+ * @typedef {{
+ *  [lang: string] : unknown
+ * }} TranslationsFile
```

Comment:
I think the difference between the `TranslationsFile` and `Translations` types is a little confusing. `TranslationsFile` is the raw data that comes from the file and `Translations` the partly-parsed data from the file, which is only fully parsed later.

IMO, `findTranslationsFile` should return its result without passing anything to `parseTranslations`. `parseTranslations` should be deleted, its should go into the `*translations()` generator, and the `Translations` type should be removed.

What do you think?

Reply:
I agree. `parseTranslations` is kinda redundant here...